### DISCUSSION TITLE
help me with "python start_ofd.py" please

### DISCUSSION THREAD

**Discussion by strayshouraki** [UltimaHoarder/UltimaScraper#1487] [Q&A]
First I installed the python3.9.x version, and then I downloaded the latest version of this software, and then I successfully installed the files in requirements.txt, but at this time I clicked on start and nothing happened. Run "python" in cmd "start_ofd.py" command and then press enter, there is no response, why is this? The next step is stuck because of this step.
![image](https://user-images.githubusercontent.com/83946918/126075268-d94d2d5a-6e44-4d89-819c-f3ac8f560d56.png)

**Comment by UltimaHoarder** [comment]
Type Python and you'll see the default version, it should say 3.9
